defaults/docker_config.yaml

```diff
@@ -2,3 +2,5 @@ user_credentials_path: '~/.ssh/scylla-test'
 logs_transport: docker
 use_preinstalled_scylla: true
 docker_image: ''
+disable_argus: true
+append_scylla_args: '--blocked-reactor-notify-ms 100 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc --abort-on-internal-error 1 --abort-on-ebadf 1 --enable-sstable-key-validation 1 --smp 2 --memory 2G'
```

Comment:
this isn't related to "improvement(config): Add a configuration option to disable argus"

Reply:
I have removed it.